lib/pure/math.nim

```diff
@@ -155,62 +188,89 @@ else: # JS
   proc ln*(x: float64): float64 {.importc: "Math.log", nodecl.}
 
 proc log*[T: SomeFloat](x, base: T): T =
-  ## Computes the logarithm ``base`` of ``x``
+  ## Computes the logarithm of ``x`` to base ``base``.
+  ## .. code-block:: nim
+  ##  echo log(9.0, 3.0) ## 2.0
   ln(x) / ln(base)
 
 when not defined(JS): # C
   proc log10*(x: float32): float32 {.importc: "log10f", header: "<math.h>".}
   proc log10*(x: float64): float64 {.importc: "log10", header: "<math.h>".}
-    ## Computes the common logarithm (base 10) of `x`
+    ## Computes the common logarithm (base 10) of `x`.
```

Comment:
Here and afterwards single backticks are used, which produces italic in RST. It should be double backticks to produce code/verbatim (like in line 191, for example).

Since we're improving the documentation, this might be the right time to fix all of these too.

Reply:
Ok, I will check and change these and eventually others.